packages/deploy/test/validator.test.ts

```diff
@@ -40,11 +40,29 @@ workflows:
 
   results = parseAndValidate(doc);
 
-  t.truthy(findError(results.errors, 'duplicate key: workflow-one'));
-
-  t.truthy(findError(results.errors, 'duplicate key: foo'));
+  t.truthy(findError(results.errors, 'duplicate workflow key: workflow-one')); 
 
   t.truthy(findError(results.errors, 'jobs: must be a map'));
+  
+  doc = `
```

Comment:
I had never seen these unit tests before and they are not well structured :scream: 

I won't ask you to fix that - you've done the right thing by sticking to the established pattern.

But could you please add a little comment at the top of this `doc` declaration, something like `disallow two jobs of the same name in the same workflow`. Just to make it clear to the next developer what's actually being tested.

Reply:
Yes, I think there should have been different tests for each case. For now, I have followed the previous pattern, but I'll be happy to help if there are any changes needed.